docs/iceberg-rest-service.md

```diff
@@ -23,11 +23,19 @@ Builds with Apache Iceberg `1.3.1`. The Apache Iceberg table format version is `
 Builds with Hadoop 2.10.x, there may compatibility issue when accessing Hadoop 3.x clusters.
 :::
 
-## How to start the Gravitino Iceberg REST catalog service
+## Gravitino Iceberg REST catalog service Configurations
```

Comment:
Heading should use sentence case and it should be "service configuration"

Reply:
done

Reply:
configuration not configurations